fooBar.php

```diff
@@ -0,0 +1,30 @@
+<?php
+class FooBar
+{
+    function checkFooBar($number)
+    {
+        $result = "";
+        if ((is_int($number) || ctype_digit($number)) && (int)$number > 0)
+        {
+            if (($number % 3) == 0)
+            {
+                $result = "Foo";
+            }
+            if (($number % 5) == 0)
+            {
+                $result .= "Bar";
+            }
+            if ((($number % 5) !== 0) && (($number % 3) !== 0))
+            {
+                $result = strval($number);
+            }
+        }
+        else
+        {
+            throw new Exception();

```

Comment:
This exception is too general, You need to find one which relates to input arguments.